packages/core/core-flows/src/order/workflows/create-return.ts

```diff
@@ -318,22 +319,8 @@ export const createReturnOrderWorkflow = createWorkflow(
       prepareFulfillmentData
     )
 
-    const fulfillment = createFulfillmentWorkflow.runAsStep(fulfillmentData)
+    createFulfillmentWorkflow.runAsStep(fulfillmentData)
 
     // TODO call the createReturn from the fulfillment provider
-
-    const link = transform(
```

Comment:
comment: removed from here since we link order <> fulfillment in the `createFulfillmentWorkflow` called aabove